dev/CommonStyles/TestUI/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,684 @@
+                    <AppBarToggleButton Icon="Volume" Label="Volume"/>
+                    <CommandBar.SecondaryCommands>
+                        <AppBarButton Label="Like"/>
+                        <AppBarButton Label="Dislike"/>
+                        <AppBarToggleButton Label="Toggle"/>
+                    </CommandBar.SecondaryCommands>
+                    <CommandBar.Content>
+                        <TextBlock Text="Hello World" Margin="12"/>
+                    </CommandBar.Content>
+                </CommandBar>
+            </Grid>
+
+            <TextBlock Grid.Row="12" Text="Flyout" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="12" Style="{StaticResource LeftColumn}">
+                <Button Content="Click me">
+                    <Button.Flyout>
+                        <Flyout>
+                            <TextBlock Text="Hello World"/>
+                        </Flyout>
+                    </Button.Flyout>
+                    <Button.ContextFlyout>
+                        <Flyout>
+                            <TextBlock Text="Hello World" MinHeight="300" MinWidth="300"/>
+                        </Flyout>
+                    </Button.ContextFlyout>
+                </Button>
+            </Grid>
+            <Grid Grid.Row="12" Style="{StaticResource RightColumn}">
+                <Button Content="Click me">
+                    <Button.Flyout>
+                        <Flyout>
+                            <Flyout.FlyoutPresenterStyle>
+                                <Style TargetType="FlyoutPresenter">
+                                    <Setter Property="CornerRadius" Value="6"/>
+                                </Style>
+                            </Flyout.FlyoutPresenterStyle>
+                            <TextBlock Text="Hello World"/>
+                        </Flyout>
+                    </Button.Flyout>
+                    <Button.ContextFlyout>
+                        <Flyout>
+                            <Flyout.FlyoutPresenterStyle>
+                                <Style TargetType="FlyoutPresenter">
+                                    <Setter Property="CornerRadius" Value="6"/>
+                                </Style>
+                            </Flyout.FlyoutPresenterStyle>
+                            <TextBlock Text="Hello World" MinHeight="300" MinWidth="300"/>
+                        </Flyout>
+                    </Button.ContextFlyout>
+                </Button>
+            </Grid>
+
+            <TextBlock Grid.Row="13" Text="MenuFlyout" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="13" Style="{StaticResource LeftColumn}">
+                <Button Content="Click me">
+                    <Button.Flyout>
+                        <MenuFlyout>
+                            <MenuFlyoutItem Text="Share" >
+                                <MenuFlyoutItem.Icon>
+                                    <FontIcon Glyph="&#xE72D;" />
```

Comment:
Is this a glyph that doesn't exist in the SymbolIcon enum?  Curious why you went with a FontIcon for this one.

Reply:
This MenuFlyout is an exact copy of what's in the XAML Controls Gallery (it's the 4th configuration show). So I guess the question is... why isn't the XAML Controls Gallery using SymbolIcon? Should we update that?

Reply:
That is a good question and one that I don't have an answer to.  I haven't done anything with the code behind the XAML Controls Gallery before.  Would be curious to hear the reason, though obviously no need to block this checkin on that point.

Reply:
The SymbolIcon enumeration only has a subset of Segoe icons: https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Symbol#remarks at this point, it might be not updated anymore and FontIcon is preferred instead.